trunk/SUAVE/Methods/Performance/V_n_diagram.py

```diff
@@ -0,0 +1,477 @@
+    # Post-processing the V-n diagram
+    # ----------------------------------------------  
+    plotting_diagram(Uref, Uref_dive, Vc, Vd, load_factors_pos, load_factors_neg, airspeeds_pos, airspeeds_neg, \
+                     gust_load_factors_pos, gust_load_factors_neg, weight, altitude, vehicle)
+  
+    return 
+
+
+#------------------------------------------------------------------------------------------------------
+#USEFUL FUNCTIONS
+#------------------------------------------------------------------------------------------------------
+
+#-----------------------------------------
+# Calculate Stall and maneuver speeds
+#-----------------------------------------
+def stall_maneuver_speeds(airspeeds_pos, airspeeds_neg, weight, rho, reference_area, maximum_lift_coefficient, \
+                          minimum_lift_coefficient, load_factors_pos, load_factors_neg):
+
+    # Stall speeds
+    airspeeds_pos[1] = (2 * weight / (rho * reference_area * maximum_lift_coefficient)) ** 0.5
+    airspeeds_neg[1] = (2 * weight / (rho * reference_area * abs(minimum_lift_coefficient))) ** 0.5
+    airspeeds_pos[0] = airspeeds_pos[1]
+    airspeeds_neg[0] = airspeeds_neg[1]
+
+    # Maneuver speeds
+    airspeeds_pos[2] = airspeeds_pos[1] * load_factors_pos[2] ** 0.5
+    airspeeds_neg[2] = (2 * weight * abs(load_factors_neg[2]) / (rho * reference_area * \
+                                                                 abs(minimum_lift_coefficient))) ** 0.5
+
+    return airspeeds_pos, airspeeds_neg
+#------------------------------------------------------------------------------------------------------------
+
+#----------------------------------------
+# Calculate Stall line load factors
+#----------------------------------------
+def stall_line(load_factors, airspeeds, upper_bound, lower_bound, Num_of_points, density_ratio, \
+               rho, lift_coefficient, weight, reference_area):
+    
+    delta = (airspeeds[upper_bound] - airspeeds[lower_bound]) / (Num_of_points + 1)     # Step size
+    for i in range(Num_of_points):           
+        coef = lower_bound + i + 1
+        airspeeds = np.concatenate((airspeeds[:coef], [airspeeds[lower_bound] + (i + 1) * delta], airspeeds[coef:]))
+        Vtas = airspeeds[coef] / density_ratio * Units.knots / Units['ft/s']
+        if load_factors[1] > 0:
+            nl = 0.5 * rho * Vtas**2 * reference_area * lift_coefficient / weight
+        else:
+            nl = -0.5 * rho * Vtas**2 * reference_area * abs(lift_coefficient) / weight
+        load_factors = np.concatenate((load_factors[:coef], [nl], load_factors[coef:]))
+    
+    return load_factors, airspeeds
+#--------------------------------------------------------------------------------------------------------------
+
+#--------------------------------
+# Calculate Gust load factors
+#--------------------------------
+def Gust_loads(airspeeds, load_factors, Kg, CLa, weight, wing_loading, reference_area, rho, density_ratio, \
```

Comment:
since you are passing lots of parameters, perhaps consider passing a data structure:

```
gust_data           = Data()
gust_data.airspeeds = airspeeds
etc....
```